CMakeLists.txt

```diff
@@ -65,6 +65,8 @@ FIND_PACKAGE(PkgConfig REQUIRED)
 SET(PKG_COMMON_MODULES
   glib-2.0
   gstreamer-1.0
+  gstreamer-audio-1.0
+  gstreamer-video-1.0
```

Comment:
Do you really need gst audio/video?

Reply:
Since the debian package that is currently deployed does not provide nnstreamer.so in the default ld_library_path, gstreamer-audio-1.0 and gstreamer-video-1.0 are required for the external plugin now. After merging  https://github.com/nnsuite/nnstreamer/pull/1132, yes, gstreamer-audio-1.0 and gstreamer-video-1.0 can be excluded.

Reply:
If what you need is ```ld_library_path```, you can specify them directly in CMakeLists.txt.